src/codegen/wasm.zig

```diff
@@ -140,3 +176,29 @@ fn genCall(buf: *ArrayList(u8), decl: *Decl, inst: *Inst.Call) !void {
         .decl = target,
     });
 }
+
+fn genStore(buf: *ArrayList(u8), decl: *Decl, inst: *Inst.BinOp) !void {
+    const idx = decl.fn_link.wasm.?.getLocalidx(inst.lhs).?;
+
+    const writer = buf.writer();
+
+    // generate codegen for rhs before we store it
+    try genInst(buf, decl, inst.rhs);
+
+    // local.set
+    try writer.writeByte(0x21);
+    try leb.writeULEB128(writer, idx);
+}
+
+fn genLoad(buf: *ArrayList(u8), decl: *Decl, inst: *Inst.UnOp) !void {
+    const idx = decl.fn_link.wasm.?.getLocalidx(inst.operand).?;
+    const writer = buf.writer();
+
+    // load the local at index `idx` onto the stack
+    try writer.writeByte(0x20);
```

Comment:
I've been trying to follow the convention of adding a comment on the same line with the name of the opcode from the spec so far which for me at least reduces the number of times I need to look at the docs. 
```suggestion
    try writer.writeByte(0x20); // local.get
```
We could consider creating constants for all the instructions eventually, though I don't think that's necessary yet.

Reply:
> We could consider creating constants for all the instructions eventually, though I don't think that's necessary yet.

Agreed, as we will have quite a few more constants such as block types as well.